src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -266,8 +266,9 @@ def fall_detect(self, image=None):
                     if angle_change > self._fall_factor:
                         # insert a box that covers the whole image as a workaround
                         # to meet the expected format of the save_detections element
-                        box = [0, 0, 1, 1]
```

Comment:
@bhavikapanara simply removing the box will cause an error in the UI, but that's something we will address separately in the UI logic.

Let's add to the inference_result the pose keypoints. The intention is to store a detailed sample of a fall detection event with rich parameters that we can use in UI rendering and also in the future for model performance analysis and training.

Reply:
@ivelin I have written a code to manage inference type whether it contains bounding box or angle difference in store.py file. As I am not able to see changes in UI, it may fail in UI.

Reply:
@ivelin do you want to add all 17 pose keypoints to inference_result? At this moment inference_result contain angle difference.

Reply:
@bhavikapanara yes, let's add all keypoints that are above the required keypoint threshold. The UI can decide later how to render them. Please do it in a separate PR to keep this PR scope manageable.